atom/browser/web_contents_preferences.cc

```diff
@@ -293,12 +294,15 @@ void WebContentsPreferences::AppendCommandLineSwitches(
   if (IsEnabled(options::kWebviewTag))
     command_line->AppendSwitch(switches::kWebviewTag);
 
+  bool sandbox =
```

Comment:
Can we make this more obvious what it means, it doesn't exactly map to `--enable-sandbox`

```suggestion
  bool can_sandbox_frame =
```